mycli/main.py

```diff
@@ -730,6 +733,15 @@ def run_query(self, query, table_format=None, new_line=True):
             for line in output:
                 click.echo(line, nl=new_line)
 
+    def output_to_file(self, text):
+        output = special.get_redirected_output()
```

Comment:
Gosh, I need to drink my coffee for the day! I didn't notice it was storing the response (`use_redirected_output`) instead of computing it again. Feel free to ignore that. Only suggestion here might be to condense the conditional `mode = 'a' if output['append'] else 'w'`.